docs/website/tds/tutorial/ConfigCatalogs.html

```diff
@@ -141,12 +141,12 @@ <h4>Data Requirement for Each Service</h4>
 
   <p>You can check that a data file is recognized as "gridded" with netCDF-Java ToolsUI (available for download from the
     <a href="https://www.unidata.ucar.edu/software/netcdf-java/">netCDF-Java home page</a> or you can use
-    <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">webstart</a>).</p>
+    <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">webstart</a>).</p>
 
   <div id="expanded" class="exercise">
     <h3>Exercise: Check that the NAM Dataset is Gridded Data</h3>
     <ol>
-      <li>Open the netCDF-Java ToolsUI application [<a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">webstart</a>].
+      <li>Open the netCDF-Java ToolsUI application [<a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">webstart</a>].
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.